common/lib/xmodule/xmodule/templates/html/announcement.yaml

```diff
@@ -2,22 +2,15 @@
 metadata:
     display_name: Announcement
 data: |
-      <ol>
-      <li>
-      <h2> September 21 </h2>
+      <p>To use this template, replace the example text with your own text.</p>
+      <h2>Announcement Date</h2>
       <section class='update-description'>
-      	       <section class='primary'>
-      	       		<p> Words of encouragement! This is a short note that most students will read. </p>
-      	       		<p class='author'>Anant  Agarwal (6.002x Principal Instructor)</p>
-      	       </section>
-      	       <h3>Primary versus Secondary Updates:</h3><p>Unfortunately, the internet throws a lot of text at students, and they
-      	       do not read everything that they are given. However, many students <em>do</em> read all that they are
-      	       given, and so detailed explainations in this section will benefit the most concientious.
-      	       Any essential information should be extremely quickly summarized in the primary section for skimmers.</p>
-      	       <h3>Star Forum Poster</h3><p>
-      	       Students appreciate knowing that the course staff is reading what they post, and one of several ways
-      	       that you can do this is by acknowledging the star posters in your announcements.
-      	       </p>
-      </section>
-      </li>
-      </ol>
+                   <section class='primary'>
+                              <p>Short note that introduces the topic</p>
+                              <p class='author'>Instructor's name</p>
+                   </section>
+                   <h3>Heading for announcement 1</h3>
```

Comment:
@srpearce, given that our site name is an ```<h1>``` and the course title is an ```<h2>``` I think its a good idea to make the first heading level here an ```<h3>```.  We will likely have to make sure other components follow this convention.